### ISSUE TITLE
Utilize Tensorboard

### ISSUE THREAD

**Issue by TidalPaladin** [TidalPaladin/tiny-imagenet-demo#9]
Incorporate Tensorboard as a tool to:
- [x] Visualize the computational graph
- [x] Report validation loss / other training metrics
- [ ] Tune hyperparameters

**Comment by TidalPaladin** [comment]
Implemented in https://github.com/TidalPaladin/tiny-imagenet-demo/commit/78118162d7c649a9872449f8b7e311e41ef842a1. Hyperparameter tuning could use some work / cleanup
